cmd/release-controller/bugzilla.go

```diff
@@ -0,0 +1,77 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/golang/glog"
+	utilerrors "k8s.io/apimachinery/pkg/util/errors"
+)
+
+// syncBugzilla checks whether fixed bugs in a release had their
+// PR reviewed and approved by the QA contact for the bug
+func (c *Controller) syncBugzilla(key queueKey) error {
+	defer func() {
+		err := recover()
+		panic(err)
+	}()
+
+	release, err := c.loadReleaseForSync(key.namespace, key.name)
+	if err != nil || release == nil {
+		return err
+	}
+
+	glog.V(4).Infof("Verifying fixed bugs %s", release.Config.Name)
+
+	// check if verifyBugs publish step is enabled for release
+	verifyBugs, ok := release.Config.Publish[publishTypeBugzilla]
+	if !ok || verifyBugs.Disabled {
+		return nil
+	}
+
+	// get accepted tags
+	acceptedTags := sortedRawReleaseTags(release, releasePhaseAccepted)
+	if len(acceptedTags) == 0 {
+		return nil
+	}
+	var errs []error
+	// get latest accepted tag
+	tag := acceptedTags[0]
+	// check if tag has already been bugzilla verified
+	if tag.Annotations[releaseAnnotationBugsVerified] == "true" {
+		return nil
+	}
+
+	verifyComplete := false
+	info, err := c.getReleaseTagInfo(release.Config.Name, tag.Name, "")
+	if err != nil {
+		glog.Errorf("Failed to get release tag info for %s: %v", tag, err)
```

Comment:
If you can't get a release tag you should exit early with nil, then you don't need the else.